### PR TITLE
revendoring gardener (k/*@v0.19) and controller-runtime v0.7.1

### PR DESCRIPTION
**Pull request by MartinWeindel**
**How to categorize this PR?**

/area control-plane
/kind task
/priority normal
/platform vsphere

**What this PR does / why we need it**:
This PR revendors all the kubernetes dependencies in version v0.19.6 and controller-runtime version v0.7.1.
See https://github.com/gardener/gardener/issues/3109 for more details.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,68 +5,69 @@ go 1.15
...
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.6
+	k8s.io/apimachinery => k8s.io/apimachinery v0.19.6
+	k8s.io/apiserver => k8s.io/apiserver v0.19.6
+	k8s.io/client-go => k8s.io/client-go v0.19.6
+	k8s.io/code-generator => k8s.io/code-generator v0.19.6
+	k8s.io/component-base => k8s.io/component-base v0.19.6
 	k8s.io/helm => k8s.io/helm v2.13.1+incompatible
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.10
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.6
+	k8s.io/kube-openapi => github.com/gardener/kube-openapi v0.0.0-20201221124747-75e88872edcf // k8s-1.19
```
Comment (rfranzke) [inline go.mod:52]: Is this needed?

Comment (MartinWeindel) [reply go.mod:52]: Good question. I copied the line from the PR https://github.com/gardener/gardener/pull/3393 for consistency reasons.
see https://github.com/gardener/gardener/blob/master/go.mod#L85

This introduces this fix: https://github.com/gardener/kube-openapi/commit/75e88872edcfe2b3e9158cf29bce4d8f4459d1de ("fix handling of inlined embedded struct pointer fields")

Comment (rfranzke) [reply go.mod:52]: Yes, but do you use this in this repo? If not, I'd remove it.

Comment (MartinWeindel) [reply go.mod]: Removed as not used. Thanks for the hint.

**Review by rfranzke — APPROVED** [review]
/lgtm

**Pull request merged**